src/adapters/MeetingsJSONAdapter/controls/RosterControl.js

```diff
@@ -34,6 +34,7 @@ export default class RosterControl extends MeetingControl {
       icon: 'participant-list_28',
       text: 'Participants',
       tooltip: 'Hide participants panel',
+      pressed: true,
     };
```

Comment:
Please try to add an extra class if button has different style in order to simplify the logic.